gui/idact_app.py

```diff
@@ -11,4 +16,30 @@ def __init__(self):
         self.ui = Ui_MainWindow()
         self.ui.setupUi(self)
         self.saver = ParameterSaver()
+        self.native_args_saver = NativeArgsSaver()
+        self.add_argument_window = AddArgumentApp()
+        self.remove_argument_window = RemoveArgumentApp()
+        self.show_native_arguments_window = ShowNativeArgumentsApp()
         self.parameters = self.saver.get_map()
+
+
+class AddArgumentApp(QMainWindow):
```

Comment:
If you add more apps to this file you should rename it to "apps" or something similar or create a package Apps where you have 4 files each for one window.